### DISCUSSION TITLE
Resource Suggestion (Bitcoin Explorama)

### DISCUSSION THREAD

**Discussion by btcexplorama** [sovenor/bitcoin-rocks#8] [Ideas]
First of all: Your project is super cool and the stickers are insane!

In my opinion, Bitcoin Explorama would be a valuable addition thanks to its visual and playful learning interactions. One could either distribute the three Bitcoin Explorama interactions among the various bitcoin.rocks sections or pack it into a single section as a whole.

See http://www.bitcoin-explorama.com/

Thanks for considering.

**Comment by sovenor** [comment]
This looks like a really great resource, thank you for sharing! I will definitely add it to the website. I'm not exactly sure where yet but I will think on it and let you know.

Would you be open to adding a link to https://bitcoin.rocks on your Website Recommendations page here? https://www.bitcoin-explorama.com/websites.php?lang=en

**Comment by btcexplorama** [comment]
That sounds great, thank you!

And sure, I will add bitcoin.rocks to the recommended websites within the next few days. I was planning to do that anyway!

**Comment by sovenor** [comment]
Thanks for your patience @btcexplorama! Bitcoin Explorama has been added as the first link in the "YOU" section on the website. It's a great resource to learn about the basics of Bitcoin (and fiat money!)

You may need to clear your browser cache to see the change.

P.S. Thank you for adding bitcoin.rocks to your resources section. It looks great!

**Reply by btcexplorama** [reply]
Thank you for the update, and for including Bitcoin Explorama in the "YOU" section of bitcoin.rocks! I’m thrilled to see it's featured prominently as a resource. Looking forward to our continued collaboration!
